CONTRIBUTING.md

```diff
@@ -221,4 +221,32 @@ Once you're done, crash dump collection can be disabled by running the following
 
 ```
 reg.exe delete "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps" /f
-```
\ No newline at end of file
+```
+
+### 12) Collect wslservice time travel debugging traces
+
+To collect time travel debugging traces:
+
+1) [Install Windbg preview](https://apps.microsoft.com/store/detail/windbg-preview/9PGJGD53TN86?hl=en-us&gl=us&rtc=1)
```

Comment:
Will this work for server users?

Reply:
Unfortunately not, but I couldn't find an official way to get the .appx so for now these instructions will not work with desktop SKU's

Reply:
How do you feel about this?
<img width="575" alt="image" src="https://user-images.githubusercontent.com/102559858/225998288-bc6ecca4-4b16-4d0e-8e1a-52612bb1343f.png">

Reply:
I'd like to but winget is not available on Windows 10 so I'd rather not depend on it

Reply:
(Not on all the Windows 10 builds we support with Store WSL at least)

Reply:
@craigloewen-msft, do you have numbers on whether more users will be on win10 builds without winget, or if there are more on server?

Reply:
The vast majority of users are on client SKUs so I think the store instructions + winget instructions for server is a good compromise.